### PR TITLE
Update OS X install instructions to point to the new native installer

### PR DESCRIPTION
**Pull request by steveatinfincia**
Link points to my github release for now. Text is essentially identical to Windows instructions.

### CODE REVIEW

**Comment by steveatinfincia** [comment]
@xor-freenet if we're going to change the wording we should figure out what to say in a separate PR and change this and the Windows instructions together. 

I copied the Windows instructions here partly because the OS X instructions don't need to be different anymore, and by changing "Windows" to "OS X" in some of the existing translations they can probably be reused immediately.

**Comment by xor-freenet** [comment]
@steveatinfincia While you're totally right that fixing it once for all would be the correct thing to do from an engineering point of view, I think from a manpower point of view, we're more likely to get stuff done if we fix such things partially while a certain piece of code is being touched anyway. So if you want do fix it just for Mac, feel free to do so. You could then file a bugtracker entry to backport this to the Windows text.
If you don't feel like investing more time, and if you validated that this text does match what we say on Windows, then you hereby have my ACK anyway - "matches the Windows text" is a valid justification :)

**Comment by bertm** [comment]
I'm missing a file signature link here, I'd expect at least some SHA > 1 hash, but preferably a GPG signature.

Then, I'd expect to have this link to either download.freenetproject.org or github.com/freenet/…, linking to an installer on a third-party Github account feels a little shady.

@Thynix should we upload this release to FPI's server / Github-freenet instead, and provide a signature with your release key?

**Comment by steveatinfincia** [comment]
@bertm completely agree, thank you :)

**Comment by Thynix** [comment]
@bertm Agreed on the FPI server and Github.com/freenet, but why my signature? @steveatinfincia developed and built it; my signature doesn't seem as meaningful as that of the person who built it. Moving toward multiple signers seems preferable? Thoughts?

File: pages/download.py [hunk 1]
```diff
@@ -111,16 +111,14 @@ def get_content(self):
...
-opening it. It might take a moment to launch.
-
-**Note:** Mavericks does not ship with Java Web Start enabled. We would like
-to make a _.dmg_ for easier OS X installation but don't know how yet. If you
-are a developer and would like to join us and help it would be much
-appreciated!
+Download and run [the installer](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip)
+
+It will automatically install Freenet and other required components for you.
+When done, your default browser will automatically open up to Freenet's
```
Comment (Thynix) [inline pages/download.py]: I thought Ian pointed out it doesn't do this?

**Comment by bertm** [comment]
@Thynix My idea was that you are the release manager, and this is a release: signing it with your key would be a statement of trust.

Your arguments are valid, though. I'm not sure about the idea of having multiple signers, that might get confusing—unless we'd have one signer per project (@steveatinfincia for mactray, @Thynix for updates, …)

**Comment by bertm** [comment]
To clarify: I don't intend to ask "the release manager" to sign the application itself (as if you were the developer), just the file that's up for download (that is, provide a GPG .sig for the actual file).

File: pages/download.py (hunk 1 above)
Comment (steveatinfincia) [inline pages/download.py]: He was testing on a machine with Freenet already installed, in that case the app automatically found his installation and silently put itself in the tray (which is something I'm considering changing). During installation though the very last step is for the user to click continue which does open the web interface.

**Comment by Thynix** [comment]
Yeah, one signer per project is what I mean.

File: pages/download.py [hunk 2]
```diff
@@ -111,16 +111,14 @@ def get_content(self):
...
-it doesn't work, try [installing Java](http://www.java.com/),
-then downloading [the installer](assets/jnlp/freenet_installer.jar) and
-opening it. It might take a moment to launch.
-
-**Note:** Mavericks does not ship with Java Web Start enabled. We would like
-to make a _.dmg_ for easier OS X installation but don't know how yet. If you
-are a developer and would like to join us and help it would be much
-appreciated!
+Download and run [the installer](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip)
+([gpg signature](https://github.com/steveatinfincia/mactray/releases/download/v2.0.0/FreenetTray_2.0.0-signed.zip.sig))
```
Comment (Thynix) [inline pages/download.py]: Missing a full stop.

**Comment by Thynix** [comment]
This doesn't change the autostart behavior, which is what people will see when clicking the download button.

File: pages/download.py (hunk 2 above)
Comment (Thynix) [inline pages/download.py]: Perhaps not as part of this changeset, but it'd be nice to link to instructions on verifying signatures like [Tails does](https://tails.boum.org/download/index.en.html#index3h1).

**Comment by Thynix** [comment]
What about a news entry?

**Comment by Thynix** [comment]
Autostart is https://github.com/freenet/website/blob/next/pages/download.py#L70

**Comment by Thynix** [comment]
Deployed.

**Pull request merged**